jsk_spot_robot/jsk_spot_teleop/launch/teleop.launch

```diff
@@ -31,6 +31,10 @@
         command="load"
         file="$(find jsk_spot_teleop)/config/$(arg pad_type)_teleop_twist_joy.yaml"
         />
+    <rosparam
+        command="load"
+        file="/var/lib/robot/home_dock_id.yaml"
```

Comment:
Currently, Dock ID is specified at [config.yaml](https://github.com/sktometometo/jsk_robot/blob/b7fdc3a90851d6a0ba830052a5ef1922ba05de16/jsk_spot_robot/jsk_spot_startup/scripts/config.bash#L28) as $DOCKING_STATION_ID. Can you use this?

Reply:
OK. I'll use it.